src/reducers/authenticationReducer.js

```diff
@@ -112,6 +156,16 @@ export default (state = INITIAL_STATE, { type, payload }) => {
     case SET_USER_MODE_SUCCEEDED:
     case GET_USER_MODE_SUCCEEDED:
       return state.setIn(['user', 'settings', 'userMode'], payload);
+    case SET_SPACE_AS_FAVORITE_SUCCEEDED:
```

Comment:
This could be simplified if you separate into two (add/remove), but if you're comfortable with this way, then no problem. 👍